js/huobi.js

```diff
@@ -2751,41 +2783,38 @@ module.exports = class huobi extends Exchange {
         //
         const data = this.safeValue (response, 'data', []);
         const result = {};
+        this.networkJunctionsByTitles = {};
+        this.networkTitlesByJunctions = {};
```

Comment:
Also these names are too ambiguous, I can't tell what they store just by looking at the names